docs/queries/byrole.mdx

```diff
@@ -115,6 +116,26 @@ you can get the "Native"-tab by calling `getByRole('tab', { selected: true })`.
 To learn more about the selected state and which elements can have this state
 see [ARIA `aria-selected`](https://www.w3.org/TR/wai-aria-1.2/#aria-selected).
 
+### `busy`
+
+You can filter the returned elements by their busy state by setting `busy: true`
+or `busy: false`.
+
+For example in
+
+```html
+<body>
+  <section>
+    <div role="alert" aria-busy="false">Login failed</div>
+    <div role="alert" aria-busy="true">Error: Loading message...</div>
+  </section>
+</body>
+```
+
+you can get the "Login failed" alert by calling
+`getByRole('alert', { busy: false })`. To learn more about the busy state see
+[ARIA `aria-busy`](https://www.w3.org/TR/wai-aria-1.2/#aria-busy).
```

Comment:
To keep things consistent with the other docs changes.
Or is there a reason why the MDN article isn't referenced?

```suggestion
[ARIA `aria-busy`](https://www.w3.org/TR/wai-aria-1.2/#aria-busy) and
[MDN `aria-busy` attribute](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-busy).
```